plugins/custom-class/index.html

```diff
@@ -113,14 +123,6 @@ <h2>Output</h2>
 &lt;/code&gt;&lt;/pre&gt;</code></pre>
 </section>
 
-<section>
-	<h1>Todo</h1>
-	<ul>
-		<li>Take a function as the transformation (<a href="https://github.com/PrismJS/prism/pull/950#issuecomment-224289585">Comment in #950</a></li>
-		<li>Allow to custom tokens per language, using nested object (<a href="https://github.com/PrismJS/prism/issues/947#issuecomment-216979932">Comment in #947</a>)</li>
```

Comment:
This isn't implemented, is it?

Reply:
Maybe it doesn't matter... this can be implemented with a function.

Reply:
That's the exact thought I also had.
It's really easy to implement this using a function, so I considered it done. Plus, we can still add it in the future.